onnxruntime/core/providers/openvino/openvino_execution_provider.cc

```diff
@@ -494,24 +493,14 @@ static bool IsTypeSupported(const NodeArg* node_arg, bool is_initializer, const
 
 static bool IsNodeSupported(const std::map<std::string, std::set<std::string>>& op_map,
                             const onnxruntime::GraphViewer& graph_viewer,
-                            const NodeIndex node_idx) {
+                            const NodeIndex node_idx, std::string& device_id) {
   const auto& node = graph_viewer.GetNode(node_idx);
   const auto& optype = node->OpType();
 
   if (openvino_ep::backend_utils::IsDebugEnabled())
     std::cout << "Node " << optype << std::endl;
   const auto& domain = node->Domain();
 
-  std::string device_id = "CPU";
-
-#if defined(OPENVINO_CONFIG_GPU_FP32) || defined(OPENVINO_CONFIG_GPU_FP16)
-  device_id = "GPU";
-#endif
-
-#if defined(OPENVINO_CONFIG_MYRIAD) || defined(OPENVINO_CONFIG_VAD_M)
```

Comment:
I was using the device_id as VPU and checking for conditions because myriad and hddl have generally same conditions. If we change it to MYRIAD and HDDL i think we need to make few changes where it's being checked. Instead of if(device == "VPU") we have to change it to "MYRIAD" || "HDDL"